vaadin-chip-combobox/src/main/java/software/xdev/vaadin/chips/ChipComboBox.java

```diff
@@ -337,5 +349,13 @@ public FlexLayout getChipsContainer()
 	{
 		return this.chipsContainer;
 	}
+
+	@Override
+	public void setItems(final Collection<T> items)
+	{
+		final ArrayList<T> listItems = new ArrayList<>();
+		listItems.addAll(items);
+		this.withAllAvailableItems(listItems, true);
```

Comment:
This can be a one liner:
``this.withAllAvailableItems(new ArrayList<>(items));``